src/wizard/javascript/performance-onboarding/javascript/1.install.md

```diff
@@ -0,0 +1,18 @@
+---
+name: JavaScript
+doc_link: https://docs.sentry.io/platforms/javascript/performance/
```

Comment:
I'm not sure if this is correct. The enable tracing snippet on the performance page is slightly different. This identical snippet is found at: 
https://docs.sentry.io/platforms/javascript/#install

This applies to all of these files.

Reply:
@imatwawana I plan to use the `doc_link` attribute to place a link to the Performance setup pages on the Performance onboarding checklist.

Although the content is slightly different here, I wanted to provide the complete setup instructions without assuming the user has installed any dependencies beforehand.